src/app/stream-deck/util/deferred.ts

```diff
@@ -0,0 +1,19 @@
+export class DeferredPromise {
```

Comment:
Document this? I'm not clear what it's for

Reply:
I added a description about the use in the notification / selection flow